packages/create-next-app/README.md

```diff
@@ -39,6 +39,7 @@ pnpm create next-app blog-app
 - **--example-path &lt;path-to-example&gt;** - In a rare case, your GitHub URL might contain a branch name with a slash (e.g. bug/fix-1) and the path to the example (e.g. foo/bar). In this case, you must specify the path to the example separately: `--example-path foo/bar`
 - **--use-npm** - Explicitly tell the CLI to bootstrap the app using npm. To bootstrap using yarn we recommend to run `yarn create next-app`
 - **--use-pnpm** - Explicitly tell the CLI to bootstrap the app using pnpm. To bootstrap using yarn we recommend running `yarn create next-app`
+- **--skip-install** - Skips dependency installation step. You can use your favorite package manager.
```

Comment:
I'm thinking this should be `--no-install`.

Also, is it still needed now that we have `--use-pnpm` which satisfies #32367?

Reply:
IMHO, It's better than adding a flag for each major package manager.